src/jit/emit.cpp

```diff
@@ -1337,7 +1337,7 @@ void* emitter::emitAllocInstr(size_t sz, emitAttr opsz)
 
     emitInsCount++;
 
-#if defined(DEBUG) || defined(LATE_DISASM)
```

Comment:
Can you make sure a RELEASE build with LATE_DISASM defined still builds and works?

Reply:
OK, I tried defining LATE_DISASM and it gets 11 unresolved symbols all related to DIS.
I suppose it I actually tried linking the disassembler library it work work.

Reply:
Ok linking also works if I change jit.nativeproj:
    <EnableLateDisasm Condition="'$(BuildArchitecture)' != 'arm' and '$(BuildProjectName)' != 'CoreSys'">true</EnableLateDisasm>